templates/CRM/Event/Form/SearchEvent.tpl

```diff
@@ -27,11 +27,13 @@
  <h3>{ts}Find Events{/ts}</h3>
   <table class="form-layout">
     <tr class="crm-event-searchevent-form-block-title">
-        <td>{$form.title.html|crmAddClass:twenty}
-             <div class="description font-italic">
-                    {ts}Complete OR partial Event name.{/ts}
-             </div>
-             <div style="height: auto; vertical-align: bottom">{$form.eventsByDates.html}</div>
+        <td>
+          <label>{ts}Event Nanme{/ts}</label>
```

Comment:
Typo - should be "Event Name".  Can you get this from $form.title.label?

Reply:
@mattwire Fixed at https://github.com/civicrm/civicrm-core/pull/11457/commits/92164dec82f17a024f9f9698469e3bce55cbc444